test/modules/EdgeHubRestartTester/src/Program.cs

```diff
@@ -126,7 +105,8 @@ static async Task<int> MainAsync()
         }
 
         static async Task<Tuple<DateTime, HttpStatusCode>> RestartEdgeHub(
```

Comment:
Please update this method to use docker or any other approach to support offline scenario before working on report generation.

Reply:
This is a bigger task than I want to tackle right now. 
Would it be alright I if I get all the refactoring + verification parth in first? then revisit the restart?